modules/adding-new-extension-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-new-extension-dynamic-plugin_{context}"]
+= Adding a new extension to your plug-in
+You can add extensions to your plug-in that are loaded at runtime. The following procedure adds an example extension to your plug-in.
+
+[id="adding-tab-to-pods-page"]
```

Comment:
And then cutting this H2.

Reply:
I think I am going to make extension a concept just bc we will add in more later once done.